src/components/OcDrop.vue

```diff
@@ -113,14 +127,40 @@ export default {
         hideAll({ exclude: instance })
       },
       content,
-    })
+    }
+
+    if (this.target) {
+      config.triggerTarget = this.toggle
+        ? document.querySelector(this.toggle)
+        : this.$el.previousElementSibling
+    }
+
+    this.tippy = tippy(to, config)
   },
   methods: {
     $_ocDrop_close() {
       if (this.closeOnClick) {
         this.tippy.hide()
       }
     },
+
+    /**
+     * Programatically show the drop
+     *
+     * @public
+     */
+    $_ocDrop_show(duration) {
```

Comment:
Since this is public `show` as method name would be fine, right?

Reply:
I was thinking about this in a scope of a mixin where we also did this pattern. In the Web PR I am accessing it via ref but TBH I am not sure if the method can in any way leak out of the component

Reply:
For a mixin the namespacing makes sense, because you can't see where the method comes from when calling it. For calling it via a ref, you know where it comes from, so I personally don't need the namespacing.

Not sure either if the method can leak out in any way, but it is intentionally public, so I think we don't need to think about that, do we? `Intentionally public` evaluates to `drop the '$_'` in my mind. 🤷